keys_storage.go

```diff
@@ -72,6 +95,58 @@ func (s *KeysStorageInMemory) DeleteMk(pubKey Key, msgNum uint) error {
 	return nil
 }
 
+// See KeysStorage.
```

Comment:
Comments before functions are used in GoDoc autogenerated documentation, so better write human-readable explanation here :)